ui/components/config/provider-form.tsx

```
@@ -0,0 +1,834 @@
+                                className={`flex-1 ${
+                                  keysRequired && key.value.trim() === ""
+                                    ? "border-destructive"
+                                    : ""
+                                }`}
+                              />
+                            </div>
+                            <div className="space-y-2">
+                              <div className="flex items-center gap-2">
+                                <label className="text-sm font-medium">
+                                  Weight
+                                </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)